### ISSUE TITLE
Please define text cursor color

### ISSUE THREAD

**Issue by Mekk** [caffo/monotropic-theme#4]
In short: monotropic doesn't define text cursor color, and inherites whatever color is set before it's loaded. Could you explicitely define that grey color for the text cursor?

My case: I switch between twilight  (my preferable dark theme, which I use on the evening, or during cloudy days) and monotropic (to which I switch whenever I face stronger sunlight). Twilight sets text cursor to yellow. Monotropic doesn't override it, so after switching the theme I get white background and yellow, almost invisible, cursor.

**Comment by caffo** [comment]
@Mekk Thanks for reporting it. Will fix that soon.

**Comment by caffo** [comment]
@Mekk Isn't that what I'm doing here? https://github.com/caffo/monotropic-theme/blob/b46e94a712e01cebe69a6f7d950e91d7c7dd1b66/monotropic-theme.el#L36

**Comment by Mekk** [comment]
I must say I don't understand that line syntax and I am not sure what is it expected to do…

Still, I tried reproducing the case in isolation and it seems difficult. When I tried the following (test on emacs -q, initial dirs are from my elpa storage):
~~~
(add-to-list 'load-path 
	     "/home/marcink/DEV_hg/emacs/elpa/twilight-theme-20120412.803")
(add-to-list 'load-path 
	     "/home/marcink/DEV_hg/emacs/elpa/monotropic-theme-20180218.357")

(require 'twilight-theme)
(require 'monotropic-theme)

(load-theme 'twilight 't)
(disable-theme 'twilight)
(load-theme 'monotropic 't)
~~~
the final effect was OK.  But when I added even more:
~~~
(add-to-list 'load-path 
	     "/home/marcink/DEV_hg/emacs/elpa/twilight-theme-20120412.803")
(add-to-list 'load-path 
	     "/home/marcink/DEV_hg/emacs/elpa/monotropic-theme-20180218.357")

(require 'twilight-theme)
(require 'monotropic-theme)

(load-theme 'twilight 't)
(disable-theme 'twilight)
(load-theme 'monotropic 't)

(disable-theme 'twilight)
(load-theme 'twilight 't)
~~~
suddenly I got twilight with black cursor.

Haven't yet found which sequence leads to opposite.
